docs/index.asciidoc

```diff
@@ -852,9 +852,9 @@ not also set this field. That will raise an error at startup
   * Default value is `nil`
 
 Set which ingest pipeline you wish to execute for an event. You can also use
-event dependent configuration here like `pipeline =>
-"%{[@metadata][pipeline]}"`. The pipeline parameter won't be set if the value
-resolves to empty string ("").
+event dependent configuration here like `pipeline => "%{[@metadata][pipeline]}"`.
+If `pipeline` is not set, the plugin takes the value from `[@metadata][target_ingest_pipeline]`.
+The pipeline parameter won't be set if the value resolves to empty string ("").
```

Comment:
Since this is Tech Preview, I am wary of injecting it directly into the docs but I like your other improvements.

```suggestion
event dependent configuration here like `pipeline => "%{[@metadata][pipeline]}"`.
The pipeline parameter won't be set if the value resolves to empty string ("").
```